application/ui/widgets/eh-widgets-dir/eh-widgets/src/page/bundle-group/modal-contact-us-information/ModalContactUsInformation.js

```diff
@@ -0,0 +1,66 @@
+import {Button, Modal} from "carbon-components-react"
+import ReactDOM from "react-dom"
+import {useState} from "react"
+import "./modal-contact-us-information.scss"
+import i18n from "../../../i18n"
+
+const ContactUsInfo = ({bundleGroup, children}) => {
+  return (
+      <div className="Modal-contact-us-code">
+
+          <iframe src={bundleGroup.contactUrl} style={{width:'100%',height:'400px'}}/>
```

Comment:
Please add a title to satisfy eslint so we can keep the build clean. 
  _Line 11:11:  <iframe> elements must have a unique title property  jsx-a11y/iframe-has-title_

Reply:
Added Title attribute for iframe tags